src/resources/i18n/translations/en.json

```diff
@@ -1,378 +1,378 @@
+	"Do you want to add this episode or clip": "Add this episode or clip?",
+	"Do you want to resume ": "Resume?",
+	"Do you want to update the download limit for all of your currently subscribed podcasts": "Update the download limit for all subscribed podcasts?",
 	"Done": "Done",
-	"Download in progress": "Download in progress",
+	"Download in progress": "Downloading…",
 	"Download limit max": "Download limit max",
 	"Download limit off": "Download limit off",
 	"Download limit on": "Download limit on",
 	"Download limit": "Download limit",
 	"Download": "Download",
 	"Downloaded": "Downloaded",
-	"Downloading Episode": "Downloading Episode",
-	"Downloading": "Downloading",
-	"Downloads in progress": "Downloads in progress",
+	"Downloading Episode": "Downloading episode…",
+	"Downloading": "Downloading…",
+	"Downloads in progress": "Downloading…",
 	"Downloads": "Downloads",
-	"Edit Clip": "Edit Clip",
-	"Edit My Profile": "Edit My Profile",
+	"Edit Clip": "Edit clip",
+	"Edit My Profile": "Edit profile",
 	"Edit Playlist": "Edit Playlist",
-	"Edit clips and playlists": "- Edit clips and playlists",
+	"Edit clips and playlists": "— Edit clips and playlists",
 	"Edit": "Edit",
-	"Email": "Email",
-	"Enable LNPay": "Enable LNPay",
-	"Enable Lightning Pay switch description": "Enable this experimental feature to connect a Bitcoin Wallet wallet to Podverse, and support podcasts through Streaming Sats and Boost!",
-	"End of results": "End of results",
-	"End time cannot be equal to 0": "End time cannot be equal to 0.",
+	"Email": "E-mail",
+	"Enable LNPay": "Turn on LNPay",
+	"Enable Lightning Pay switch description": "Turn on this experimental feature to add a ₿ wallet to support podcasts through streaming Sats and boost.",
+	"End of results": "No more results",
+	"End time cannot be equal to 0": "End time cannot be \"0\".",
 	"End time": "End time",
-	"Episode Clips": "Episode Clips",
-	"Episode Funding Links": "Episode Funding Links",
-	"Episode Not Found": "Episode Not Found",
-	"Episode Summary": "Episode Summary",
-	"Episode is loading": "Episode is loading",
+	"Episode Clips": "Episode clips",
+	"Episode Funding Links": "Episode funding links",
+	"Episode Not Found": "Could not find the episode",
+	"Episode Summary": "Summary",
+	"Episode is loading": "Loading episode…",
 	"Episode": "Episode",
 	"Episodes from this podcast will start playback from this time": "Episodes from this podcast will start playback from this time",
 	"Episodes": "Episodes",
-	"Error Reporting Disabled": "Error Reporting Disabled",
-	"Error Reporting Enabled": "Error Reporting Enabled",
-	"Error Reporting subtext": "Error reports are anonymous and help us identify and fix bugs faster.",
-	"Error Setting External Storage": "Error Setting External Storage",
+	"Error Reporting Disabled": "Error reporting off",
+	"Error Reporting Enabled": "Error reporting on",
```

Comment:
Everything here is now "… on" or "… off". No enabled or disabled. Similarly, "turn on", "turn off".